src/Sulu/Bundle/TestBundle/Testing/SuluTestCase.php

```diff
@@ -11,311 +11,14 @@
 
 namespace Sulu\Bundle\TestBundle\Testing;
 
-use Doctrine\Common\DataFixtures\Executor\ORMExecutor;
-use Doctrine\Common\DataFixtures\ProxyReferenceRepository;
-use Doctrine\Common\DataFixtures\Purger\ORMPurger;
-use Doctrine\ORM\EntityManager;
-use PHPCR\ImportUUIDBehaviorInterface;
-use PHPCR\SessionInterface;
-use PHPCR\Util\NodeHelper;
-use Sulu\Bundle\DocumentManagerBundle\Initializer\Initializer;
-use Sulu\Bundle\PageBundle\Document\HomeDocument;
-use Sulu\Component\Content\Document\WorkflowStage;
-use Symfony\Bundle\FrameworkBundle\Client;
-use Symfony\Component\Filesystem\Filesystem;
-use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
 
-abstract class SuluTestCase extends KernelTestCase
+abstract class SuluTestCase extends WebTestCase
 {
-    private static $workspaceInitialized = false;
-
-    /**
-     * @var PHPCRImporter
-     */
-    protected $importer;
```

Comment:
Is used only in the NodeControllerTest so I did only implement it there

Reply:
Sounds reasonable to me. :+1: